src/main/java/jenkins/plugins/slack/SlackNotifier.java

```diff
@@ -584,8 +605,8 @@ public boolean configure(StaplerRequest req, JSONObject formData) {
             return true;
         }
 
-        SlackService getSlackService(final String baseUrl, final String teamDomain, final String authToken, final String authTokenCredentialId, final boolean botUser, final String room) {
-            return new StandardSlackService(baseUrl, teamDomain, authToken, authTokenCredentialId, botUser, room);
+        SlackService getSlackService(final String baseUrl, final String teamDomain, final String authTokenCredentialId, final boolean botUser, final String room) {
+            return new StandardSlackService(baseUrl, teamDomain, null, authTokenCredentialId, botUser, room);
```

Comment:
Oh please, no don't ship a null object! :cry:

Reply:
added another method, but StandardSlackService is still called from freestyle and pipeline with token currently